core/third_party_libs/pue/pue-client.php

```diff
@@ -1643,11 +1662,19 @@ public function checkForUpdates()
         {
             $state = get_site_option($this->optionName);
 
+            // If this is an add-on, only call home if there is an update available for it.
+            if( $this->slug !== 'event-espresso-core-reg' && !empty($state->latestVersion) ) {
+                if( version_compare($this->_installed_version, $state->latestVersion, '>=') ) {
+                    return;
+                }
+            }
```

Comment:
ultimately comes down to a simple yet brilliant fix... nice job 🥇

Reply:
Thanks :)